### ISSUE TITLE
Views: allow setting the "Items to display" value to "unlimited".

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1460]
This is basically the same as #1459 but for Views instead the fields.

Here's what the UI looks like now:

![views-items_to_display](https://cloud.githubusercontent.com/assets/2423362/12020189/89ac0dc8-adcd-11e5-8374-9d65d07f7cc2.png)

I think we should apply the same limited/unlimited drop-down menu pattern here as well, but with the default being set to "limited" and "10" (so that we don't change the current default). The end goal is to allow users to select "unlimited".

**Comment by BWPanda** [comment]
I agree, but instead of using the word 'Unlimited' here I think we should use the word 'All'. Views shows existing content, of which there will always be a finite amount (so 'unlimited' doesn't really make sense). Whereas when adding field values, you can technically sit there for all eternity adding more (so 'unlimited' is more accurate there).
